### DISCUSSION TITLE
Project Setting for default script folder / remember last folder used for scripts

### DISCUSSION THREAD

**Discussion by Hatchet2k4** [godotengine/godot-proposals#9246] [Editor]
Using Godot 4.2.1. Whenever I go to create a new script, it always wants to add it to my scenes folder, however I very much prefer to have my source in a separate source folder as that's cleaner for me when using VS Code, as I also have code that's not necessarily attached to any scenes or nodes. However, this means I need to change the folder every time I add/attach a new script. 

I propose either adding a project setting to specify a default folder for scripts, or for Godot to remember (per project) the last folder I used for a script.

**Comment by AThousandShips** [comment]
This is because it follows the [best practices](https://docs.godotengine.org/en/stable/tutorials/best_practices/project_organization.html#organization) on project organization I'd say

**Reply by AThousandShips** [reply]
See also:
* https://github.com/godotengine/godot-proposals/issues/1324
* https://github.com/godotengine/godot/issues/30975
* https://github.com/godotengine/godot/issues/15231
* https://github.com/godotengine/godot-proposals/issues/876
